core/src/main/java/org/mskcc/cbio/portal/dao/DaoGeneOptimized.java

```diff
@@ -95,11 +95,12 @@ private synchronized void fillCache() {
                 if (gene==null) {
                     ProgressMonitor.logWarning(line+" in config file [resources" + GENE_SYMBOL_DISAMBIGUATION_FILE +
                             "]is not valid. You should either update this file or update the `gene` and `gene_alias` tables to fix this.");
+                } else {
+                    disambiguousGenes.put(parts[0], gene);
                 }
-                disambiguousGenes.put(parts[0], gene);
             }
             in.close();
-        } catch(IOException e) {
+        } catch(Exception e) {
```

Comment:
Just curious, are Exceptions besides IOException thrown here?

Reply:
There was a RuntimeException, but should be fixed with the fix above.

Just trying to cast a wider net since not being able to load the disambiguate file is not critical issue and shouldn't bring down the application.

Reply:
No it was b/c the entrez IDs in the file are human IDs and therefore do not exist in mouse genes, ie. gene==NULL. As a result, `disambiguousGenes.put(parts[0], gene);` throws runtime exception.

Reply:
I am ok to revert this one as long as we get the change to line 99 in :)